File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        n = len(nums)
        
        for i in range(n):
            if i != nums[i]:
                return i
            elif nums[n-1] == n - 1:
                return n

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int esum = 0;
        int asum = 0;
        for(int i = 0; i < n;i++){
            esum = esum + i;
            asum = asum + nums[i];
        }
        esum = esum + n;
        int result = esum - asum;
        return result;
    }
}